js-miniapp-sample/src/pages/landing.js

```diff
@@ -22,14 +22,18 @@ const useStyles = makeStyles((theme) => ({
       lineHeight: 1.5,
     },
   },
+  info: {
+    fontSize: 16,
+  },
 }));
 
 const Landing = () => {
   const classes = useStyles();
   return (
     <GreyCard className={classes.card}>
       <CardContent className={classes.content}>
-        <p>This is a Demo App of Mini App JS SDK</p>
+        <p>Demo Mini App JS SDK</p>
+        <p className={classes.info}>Platform: {MiniApp.getPlatform()}</p>
```

Comment:
😉 I was thinking to include some big designs for the Home page😛 . But I guess this is ok for Sample app.

Reply:
That design could come from another PR. Please feel free to update it!